### ISSUE TITLE
Decorative images, lightbox links and screenreaders

### ISSUE THREAD

**Issue by FayCross** [thexerteproject/xerteonlinetoolkits#1373]
Following on from a [post on the forum](https://xerte.org.uk/index.php/en/forum/accessibility/3455-decorative-images-still-being-read-out-by-safari-voiceover-on-iphone), should we automatically prevent images without alt text from being wrapped in a link that opens the image in a lightbox? If the image is decorative only, would it ever be worth opening it in a lightbox? As Ron has said in the forum, there are reasons why allowing images to open in a lightbox is valuable so it would be a shame if people started to turn this off for whole projects because it is a confusing for decorative images.

**Comment by ronm123** [comment]
@FayCross although Lilian's forum post was about decorative text and what a screen reader then reads I think it's important to be clear about these different things separately. e.g.

If the alt text is left empty because the image is decorative then the current code is correct in that the alt tag is empty and will not be read by the screen reader. But as we know there are other factors involved here:
1 in Lilian's example it wasn't a problem with empty alt text just that the lightbox link was being read
2 not all authors take the time to add alt text and unfortunately that doesn't necessarily mean the image is only decorative and/or shouldn't open in a lightbox
3 whether the image should open in a lightbox or not should be a separate decision and any solution needs to cover images added via the wizard and also the editor toolbar

I can see that we could prevent images without alt text from being wrapped in the lightbox link but I'm not sure that's the right solution. At the moment we have project wide and page specific options for enabling lightbox or not and perhaps we should have a checkbox per image in the wizard and the toolbar so that the choice is even more granular and then deliberate for each image?
